packages/quill/src/main.ts

```diff
@@ -54,18 +54,15 @@ const createWindow = (): void => {
     height: HEIGHT,
     fullscreenable: false,
     resizable: isDev,
-    // Spectron fails randomly when the app is frameless
-    // We don't use NODE_ENV as it is controlled by electron-forge
-    frame: process.env.IS_TEST === 'true',
+    frame: false,
     transparent: true,
     webPreferences: {
       devTools: isDev,
       preload: MAIN_WINDOW_PRELOAD_WEBPACK_ENTRY,
       // For security reasons the following params should not be modified
       // https://electronjs.org/docs/tutorial/security#isolation-for-untrusted-content
       nodeIntegration: false,
-      contextIsolation: true,
-      worldSafeExecuteJavaScript: true
```

Comment:
Is this safe to remove?

Reply:
Yes. It has been removed and the behaviour is enabled by default: https://www.electronjs.org/blog/electron-14-0#removed-worldsafeexecutejavascript